xml/cl.xml

```diff
@@ -6983,7 +6983,7 @@ server's OpenCL/api-docs repository.
                 <enum name="CL_DEVICE_INTEGER_DOT_PRODUCT_ACCELERATION_PROPERTIES_4x8BIT_PACKED_KHR"/>
             </require>
         </extension>
-        <extension name="cl_khr_semaphore" revision="0.9.1" supported="opencl" depends="CL_VERSION_1_2" ratified="opencl">
```

Comment:
Are you sure this is intentional? There are a few more extensions that already had a revision attribute, and have been reset to `1.0.0`.

Reply:
Yes this is intentional - I manually looked through the extensions while I was doing this and changed the revisions in the xml tag to match the spec. Several of the semaphore and external memory family of extension tags needed bumped to 1.0.0. Saying that, because I did this manually I still could have made a mistake so double checking these revisions are correct should be verified in review.